app/assets/javascripts/discourse/app/instance-initializers/sniff-capabilities.js

```diff
@@ -1,9 +1,6 @@
 export default {
-  name: "sniff-capabilities",
-  after: "export-application-global",
```

Comment:
Forgot to mention about this (and a few other instances of the same thing):

`"export-application-global"` (from [ember-export-application-global](https://github.com/ember-cli/ember-export-application-global)) is actually an _application_ initializer that was incorrectly run as instance initializer with the previous setup.

With that fixed now (fixing that itself could have caused issue and worth calling out), this now errors because there is no instance initializer with that name. Since application initializers are guaranteed to have already run, and `"export-application-global"` itself has no other ordering dependencies, I figured I could just remove all of these, and as far as I can tell it didn't cause any problems.